src/test/java/lv/ctco/javaschool/vote/boundary/VoteApiTest.java

```diff
@@ -194,6 +196,28 @@ void submitDailyVote_Unhappy_with_Comment() {
         verify(em, times(1)).persist(any(DailyVote.class));
     }
 
+    @Test
+    @DisplayName("GetAllEventVotesTest: Checks if method returns correct dto value")
+    void getAllEventVotesTest(){
+        EventVote ev = new EventVote();
+        Event event = new Event();
+
+        event.setEventName("testEvent");
+        ev.setComment("test");
+        ev.setMood(MoodStatus.HAPPY);
+        ev.setUser(user1);
+        ev.setEvent(event);
+        when(voteStore.getAllEventVotes())
+                .thenReturn(Collections.singletonList(ev));
+
+        List<EventVoteDto> test = voteApi.getAllEventVotes();
+        EventVoteDto testResultDto = test.get(0);
+        assertThat(testResultDto.getComment(), equalTo(ev.getComment()));
+        assertThat(testResultDto.getEventName(), equalTo(event.getEventName()));
+        assertThat(testResultDto.getMood(), equalTo(ev.getMood()));
+        assertThat(test.size(),equalTo(1));
```

Comment:
This check should be done right after line 213 (`List<EventVoteDto> test = voteApi.getAllEventVotes();`), because:
1) this check is quicker to do
2) if this check fails, there is no point in checking the `EventVoteDto`, because we cannot be sure that the item we expect is at position 0